src/reynir_correct/pattern.py

```diff
@@ -166,6 +175,28 @@ def __init__(self, ann: List[Annotation], sent: Sentence) -> None:
                 # First instance: create the class-wide pattern list
                 self.create_patterns()
 
+    def get_wordform(self, lemma, cat, variants):
+        """ Get correct wordform from BinPackage, 
+        given a set of variants """
+
+        # Get rid of argument variants in verbs:
+        variants = list( [ x for x in variants if not x.isdigit()])
```

Comment:
list() not necessary around list comprehension.

Reply:
...but the isdigit() test is done again in the for loop anyway, so it can be skipped here. Or the SKIPVARS set check added to the list comprehension filter condition. No need to duplicate the filtering.